ghost/core/test/e2e-server/services/milestones.test.js

```diff
@@ -156,7 +157,8 @@ describe('Milestones Service', function () {
     beforeEach(async function () {
         loggingStub = sinon.stub(logging, 'info');
         const now = new Date();
-        clock = sinon.useFakeTimers(now.getTime());
+        const threeMonthsAgo = now.setMonth(now.getMonth() - 3);
```

Comment:
The `setMonth` method mutates the `Date` instance, so after this line the `now` variable is actually three months ago. Obviously fine here but if someones changes the tests and tries to use the `now` value they'll get unexpected results!

Also the value of `new Date(threeMonthsAgo).getTime()` and `threeMonthsAgo` are the same! Date objects are freaking weird.

I would probably change this to something like:
```js
const threeMonthsAgo = new Date();
threeMonthsAgo.setMonth(threeMonthsAgo.getMonth() - 3);
clock = sinon.useFakeTimers(threeMonthsAgo.getTime());
```

Reply:
Moved test updates to https://github.com/TryGhost/Ghost/pull/16314